spec/src/main/doc/core/scopescontexts.asciidoc

```diff
@@ -170,10 +169,10 @@ The +@Dependent+ scope is always active.
 
 ==== Dependent objects
 
-Many instances of beans with scope +@Dependent+ belong to some other bean or Java EE component class instance and are called _dependent objects_.
+Many instances of beans with scope +@Dependent+ belong to some other bean are called _dependent objects_.
```

Comment:
I dislike this sentence somehow. Shouldn't be there "belonging" instead of "belong" ?